documentation/modules/tracing/proc-enabling-tracing-in-connect-mirror-maker-bridge-resources.adoc

```diff
@@ -129,6 +136,43 @@ spec:
 ----
 --
 
+. (Optional) If TLS authentication is configured on the OTLP endpoint, use the `OTEL_EXPORTER_OTLP_CERTIFICATE` environment variable to specify the path to a trusted certificate
+This secures communication between Kafka components and the OpenTelemetry endpoint.
++
+To provide the certificate, mount a volume containing the secret that holds the trusted certificate.  
+Unless the endpoint address is redirected from `http`, use `https`.
++
+.Example configuration for TLS
+[source,yaml,subs=attributes+]
+----
+apiVersion: {KafkaConnectApiVersion}
+kind: KafkaConnect
+metadata:
+  name: my-connect-cluster
+spec:
+  #...
+  template:
+    connectContainer:
+      env:
+        - name: OTEL_SERVICE_NAME
+          value: my-otel-service
+        - name: OTEL_EXPORTER_OTLP_ENDPOINT
+          value: "https://otlp-host:4317"
+        - name: OTEL_EXPORTER_OTLP_CERTIFICATE
+          value: "/mnt/mysecret"
```

Comment:
Is this really a directory? Or should it be a path to the certificate file?

Reply:
Yes, it has to be the path to the cert file. This is also how I tested it with the OpenShift-based example at https://github.com/strimzi/strimzi-kafka-operator/issues/9170